File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/JS/Backstage/CF/AssociationRule.js
$(document).ready(function () {
    $.ajax({
        url: '/BCF/GetAssociationRuledata_B',
        type: 'POST',
        dataType: 'json',
        success: function (data) {
            console.log(data);
        },
        error: function () {
            console.log("無法取得db");
        }
    });
    //$.ajax({
    //    url: '/BCF/CreateAssociationRuledata_B',
    //    type: 'POST',
    //    data: {
    //        AssociationRule_Title: 'test333',
    //        AssociationRule_Detail: '2015/08/30',
    //        AssociationRule_Modifier: 17,
    //        AssociationRule_Modificationdatetime: '2015/08/31'
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BCF/UpdateAssociationRuledata_B',
    //    type: 'POST',
    //    data: {
    //        AssociationRule_Title: 'test33345',
    //        AssociationRule_Detail: 'test',
    //        AssociationRule_Modifier: 16,
    //        AssociationRule_Modificationdatetime: '2015/08/31',
    //        AssociationRule_ID: 10
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
    //$.ajax({
    //    url: '/BCF/DeleteAssociationRuledata_B',
    //    type: 'POST',
    //    data: {
    //        AssociationRule_ID: 11
    //    },
    //    dataType: 'json',
    //    success: function (data) {
    //        console.log("test");
    //    },
    //    error: function () {
    //        console.log("無法取得db");
    //    }
    //});
})

File: MODERN-ART-ASSOCIATION-OF-KAOHSIUNG/Models/TableModel/AssociationRule.cs
namespace MODERN_ART_ASSOCIATION_OF_KAOHSIUNG.Models.TableModel
{
    public class AssociationRule
    {
        /// <summary>
        /// 資料表名稱
        /// </summary>
        public string TABLE_NAME { get { return "AssociationRule"; } }
        
        /// <summary>
        /// 學會規章詳細資料
        /// </summary>
        public string AssociationRule_Detail { get; set; }

        /// <summary>
        /// 學會規章編號
        /// </summary>
        public string AssociationRule_ID { get; set; }

        /// <summary>
        /// 修改時間
        /// </summary>
        public string AssociationRule_Modificationdatetime { get; set; }

        /// <summary>
        /// 修改人
        /// </summary>
        public string AssociationRule_Modifier { get; set; }

        /// <summary>
        /// 學會規章標題
        /// </summary>
        public string AssociationRule_Title { get; set; }

        public void PK_AssociationRule(string AssociationRule_ID)
        {

        }
    }
}